procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb

```diff
@@ -0,0 +1,631 @@
+{
```

Comment:
<p>So, I'd suggest using the "title" format to add a short tile and them use normal text to explain what is going on here, e.g.;</p><p> </p><pre class="ql-syntax" spellcheck="false"># Daytime Checkout

This notebook is run a series of tests to ensure the system responds as expected. 
It begins with LATISS but can also be used with the telescope components only.
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>